doc/running_on_phone.md

```diff
@@ -35,21 +42,63 @@ Files required to boot from eMMC are:
 
 ### Uploading it to the eMMC
 
--  Use the `pureflash` command to burn image into the eMMC card `sudo pureflash <PurePhone.img> /dev/sdX` **Note:** Replace `/dev/sdX` with the disk name you found for your phone after connecting it to your computer.
+####  via bootloader
+
+On linux you can use:
+1. `pureflash`
+
+Fastest command to burn image into the eMMC card `sudo pureflash <PurePhone.img> /dev/sdX` **Note:** Replace `/dev/sdX` with the disk name you found for your phone after connecting it to your computer.
+
+2. `dd` or `etcher`
+
+You can use any raw memory copy tool such as `dd` or `etcher` which will transfer whole image built to the device memory. It will take much longer because it will copy whole 16GB memory.
+dd example:
+```
+sudo dd if=./PurePhone.img of=/dev/sdb bs=1M status=progress conv=fsync                                                                                                                                                                                                    13:48:22
+```
+
+On MacOS you can use script:
+- `./tools/macflash.sh`
+
+**NOTE:** Devices in the field have direct eMMC via bootloader locked.
+
+#### via uptader utility
+
+Every device is loaded with additional update software which is capable of updating any software package onto the device, while keeping last software backup.
+Updater utility is the only normal way to update devices running in field and is used by MuditaCenter.
+
+**NOTE:** Porviding a minimum viable update package and utility for Open Source community is on our backlog. It can be created by trimming additional developemnt api from:
```

Comment:
```suggestion
**NOTE:** Providing a minimum viable update package and utility for the Open Source community is on our backlog. It can be created by trimming additional development API from:
```